roadmap.md

```diff
@@ -0,0 +1,33 @@
+# Roadmap
+
+This feature list is a peek into our roadmap. It identifies some of the significant features we are currently working on and a rough timeframe for when you can expect to see them. It is not comprehensive but is intended to provide some visibility into key investments.
+
+These features and dates are the current plans at this time and are subject to change. The **Area** column reflects the area of the product the feature aligns with most, and **State**  indicates if the feature will be in public preview or generally available.  
+
+### Currently in Private Preview
+| Area    | State          | Feature                                                |
+| ------- | -------------- | ------------------------------------------------------ |
+| Calling | Private Preview | .NET client library for Calling               |
+| Calling | Private Preview | [Anonymous interoperability with Microsoft Teams](https://docs.microsoft.com/azure/communication-services/concepts/voice-video-calling/teams-interop)   |
+
+
+### 1Q CY2021
+January - March 2021
+
+| Area    | State          | Feature                                                |
+| ------- | -------------- | ------------------------------------------------------ |
+| Calling | Public Preview | .NET client library for Calling               |
```

Comment:
Should it be renamed to UWP? Since it's actually C++ library but gives the ability to use .Net technology? @mariusu-msft